libs/payments/cart/src/lib/checkout.service.spec.ts

```diff
@@ -0,0 +1,349 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Test } from '@nestjs/testing';
+import { CheckoutService } from './checkout.service';
+import { faker } from '@faker-js/faker';
+import { ResultCartFactory } from './cart.factories';
+import {
+  StripeClient,
+  StripeConfig,
+  StripeCustomerFactory,
+  StripeInvoiceFactory,
+  StripeManager,
+  StripePaymentIntentFactory,
+  StripePaymentMethodFactory,
+  StripePromotionCodeFactory,
+  StripeResponseFactory,
+  StripeSubscriptionFactory,
+} from '@fxa/payments/stripe';
+import {
+  PayPalClient,
+  PayPalManager,
+  PaypalCustomerManager,
+  ResultPaypalCustomerFactory,
```

Comment:
tiny nit, :broken_record_emoji: but not sure if we can change these to `PayPalCustomerManager` and `ResultPayPalCustomFactory` for consistency (since we seem to be exporting both from same lib as `PayPalManager` and `PayPalClient`).

I did a quick grep of the [`@paypal/paypal-js` repo](https://github.com/search?q=repo%3Apaypal%2Fpaypal-js+paypal+language%3ATypeScript&type=code&p=2&l=TypeScript) and looks they use 
- Pascal `PayPalButtonsComponent`, `PayPalNamespace` and `getPayPalWindowNamespace()` or
- all lowercase `paypal` like `let paypal: PayPalNamespace` or `const paypalFromWindow = (window as any).paypal as PayPalNamespace;`.